modules/status/main.tf

```diff
@@ -0,0 +1,116 @@
+terraform {
+  required_providers {
+    acme = {
+      source  = "vancluever/acme"
+      version = "2.19.0"
+    }
+  }
+}
+
+locals {
+  fqdn = "${var.subdomain}.${var.root_zone_name}"
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = "status-rg"
+  location = var.location
+}
+
+# App Service (Linux Web App)
+resource "azurerm_linux_web_app" "status_app" {
```

Comment:
Where is the configuration for the targets? Or am I missing something?

Reply:
If by targets you mean the sites that are going to be monitored, I'm unsure if they can be configured declaratively.

Reply:
Hmm, well how are they going to be configured then? There is no state currently in this PR (container restart/redeployment will reset configs)

Reply:
Hmm, I'll need to add a state then

Reply:
Or could we do the configuration somehow declaratively in terraform?

Reply:
Haven't found an easy way but I agree that it would be better

Reply:
Are there some similar status things where it could be configured declaratively? Or pre-configured some how, doesn't have to be declarative

Reply:
Also doesn't this thing need a database anyway?

Reply:
Yes